.asf.yaml

```diff
@@ -31,12 +31,12 @@ github:
     rebase: false
   collaborators:
     - sullis
-    - shenyu0127
-    - tibrewalpratik17
+    - yashmayya
```

Comment:
I guess you used some kind of script to generate this list. Can you sort the list in alphabetical order? that would make it easier to actually understand who gets added and who gets removed.

Reply:
It's sorted in order of # of commits in the past year (from the output of `git shortlog --numbered --summary --since="1 year ago"`).